reposilite-backend/src/main/kotlin/com/reposilite/token/api/TokensResponse.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 dzikoysk
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.reposilite.token.api
+
+data class TokensResponse(val tokens: List<AccessTokenField>) {
+    data class AccessTokenField(
```

Comment:
Frontend and other components use just deserialized AccessToken, so there is no point to create dedicated dto at this moment for this, so just use them directly

Reply:
Hmmm, I decided to use this instead because I didn't think it would be the best idea to return the secret also but if you believe that's alright then I'll switch it to that

Reply:
Never mind, I was mistaken! Fixing and marking as resolved